File: ILoopAPI/Models/Activity.cs
using System;

namespace ILoopAPI.Models
{
    public class Activity
    {
        public int Id { get; set; }
        public string Type { get; set; }
        public int Duration { get; set; }
        public DateTime Date { get; set; }
        public string Comment { get; set; }
        public string Project { get; set; }
        public bool Status { get; set; }

        public User User{get; set;}


    }
}

File: ILoop/src/app/_models/Activity.ts
import { User } from "./User";

  export class Activity {
    id:number;
    type:string;
    duration:number;
    date:Date;
    comment:string;
    project:string;
    status:boolean;
    user:User;
  
  }
  
  export class ActivitySetStatusDTO
  {
    id:number;
    status:boolean;
  }

